examples/java/README.md

```diff
@@ -0,0 +1,115 @@
+# Connecting to AlloyDB
+
+## Before you begin
+
+1. Enable access to AlloyDB in your project by following these [instructions](https://cloud.google.com/alloydb/docs/project-enable-access)
+
+1. Create a VPC network and [configure Private Services Access for AlloyDB](https://cloud.google.com/alloydb/docs/configure-connectivity)
```

Comment:
Do they need to configure PSA first?

Reply:
no, first we need to enable the AlloyDB API https://cloud.google.com/alloydb/docs/connect-psql